docs/content/concepts/layer/control/catalog/_index.md

```diff
@@ -85,157 +88,271 @@ See [examples](#content-examples) of OSCAL catalogs.
+#### Control, ID and title
 
-{{<highlight xml "linenos=table,nowrap=false" >}}
+This control has a number of high-level data elements, including a security control identifier ("AC-1"), a title ("ACCESS CONTROL POLICY AND PROCEDURES"), the control (text) itself, supplemental guidance, control enhancements (although not in this example), and references.
+
+See the source code as currently available in [raw XML from Github](https://raw.githubusercontent.com/usnistgov/oscal-content/main/nist.gov/SP800-53/rev5/xml/NIST_SP-800-53_rev5_catalog.xml). The control is identified in code with tagging that looks *mainly* as follows, with slight simplifactions. Control AC-1 starts:
+
+{{<highlight xml "nowrap=false" >}}
 <control class="SP800-53" id="ac-1">
-  <title>Access Control Policy and Procedures</title>
-  <param id="ac-1_prm_1">
+    <title>Policy and Procedures</title>
+{{< / highlight >}}
+
+- The control is assigned a class, "SP800-53". This serves as an indicator to a downstream processor of the control's origin (with respect to the catalog or catalog type), with implications for related expectations, including expectations in detail regarding how the control and its contents may be structured.
+- The control ID is "ac-1". All `id` values are unique within the document and serve for addressing and linking. The lexical form of the `id` is designed to be consistent with its formal (conventional) code or label, which is *also* encoded within the control (in this case, "AC-1"). While such regularities are normal and to be expected, this one is a feature of this data set, not a requirement of OSCAL generally.
+
+#### Parameters
+
+The control continues with `<param>` elements. This is the first place where the electronic file definitively and explicitly captures information about the source data that is left implicit in in its published form (that is, as a document). In this case, the information captured regards options for control configuration (by users of the catalog) in detail, which will subsequently be reflected in the control text as rendered, but which are called out here as first-class objects so that OSCAL consumers can handle them in appropriate ways.
+
+A *parameter* in OSCAL (tagged as `<param>` in XML or, in JSON or YAML, as an object member of a `parameters` array) corresponds to and supports the construct called *controlled parameter value* (CPV) in NIST SP 800-53A, while not being limited exclusively to that use.
+
+*Note*: In this data, they appear with the controls in which they are used, but in OSCAL data, parameters may also be defined elsewhere, including at the group level or across a catalog, thus facilitating their consolidation and reuse.
+
+*Note also*: for succinctness, the code sampled here shows only the first four of the parameters defined for AC-1 in the document (version) it was derived from.
+
+{{<highlight xml "nowrap=false" >}}
+<param id="ac-1_prm_1">
+    <prop name="aggregates" ns="http://csrc.nist.gov/ns/rmf"
```

Comment:
Minor, non-blocking: should we talk about the addition of `ns` to properly identify and segment controls by data set owner/control framework more effectively? We added this pretty late in the game but something we might want to highlight here, or maybe better in the "Learn to Make a Catalog Tutorial" that is more involved? Either way, just food for thought. Would love your take on that.